docker-image-src/4.0/Dockerfile

```diff
@@ -22,14 +22,14 @@ RUN apt update \
     && tar --extract --file ${NEO4J_TARBALL} --directory /var/lib \
     && mv /var/lib/neo4j-* "${NEO4J_HOME}" \
     && rm ${NEO4J_TARBALL} \
+    && chown -R neo4j:neo4j "${NEO4J_HOME}" \
+    && chmod -R 777 "${NEO4J_HOME}" \
     && mv "${NEO4J_HOME}"/data /data \
     && mv "${NEO4J_HOME}"/logs /logs \
     && chown -R neo4j:neo4j /data \
-    && chmod -R 777 /data \
     && chown -R neo4j:neo4j /logs \
```

Comment:
since you've moved the `chown -R` and `chmod -R` on `${NEO4JHOME}` up and it's recursive you should be able to get rid of the separate `chown` and `chmod` on `/data` and `/logs`